devices/common_patches/h7_exti_dualcore.yaml

```diff
@@ -0,0 +1,853 @@
+# H7 EXTI module, for dual core parts
+
+EXTI:
+  _modify:
+    C1PR2,C1IMR3,C1EMR3,C1PR3:
+      access: read-write
+
+  _add:
+    C2IMR1:
+      description: EXTI interrupt mask register
+      addressOffset: 0x00C0
+      resetValue: 0xFFC00000
+      fields:
+        MR0:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 00
+          bitWidth: 1
+        MR1:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 01
+          bitWidth: 1
+        MR2:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 02
+          bitWidth: 1
+        MR3:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 03
+          bitWidth: 1
+        MR4:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 04
+          bitWidth: 1
+        MR5:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 05
+          bitWidth: 1
+        MR6:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 06
+          bitWidth: 1
+        MR7:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 07
+          bitWidth: 1
+        MR8:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 08
+          bitWidth: 1
+        MR9:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 09
```

Comment:
To avoid YAML confusion probably best to specify these numbers as just `9` instead of `09` etc (I believe by default this will just cause it to load as a string instead of an integer, rather than processing as octal, but that's still not ideal).

Reply:
Tidied all leading zeros, it was an error from the regex I used to write it all this